src/components/Documentation/Markdown/index.tsx

```diff
@@ -50,11 +50,62 @@ const Abbr: React.FC<{ children: [string] }> = ({ children }) => {
   return <Tooltip text={children[0]} />
 }
 
+const Cards: React.FC<{
+  children: Array<object | string>
```

Comment:
out of curiosity - why is it different from the above type declaration? Why is `| string`, btw?

Reply:
Newlines in markdown manifest as `"\n"` strings in `children`, so `string` needs to be considered as a possible input with custom components that consume Markdown.
`children` is a non-array in component with <= 1 children, so that case has to be considered for when manipulating `children` with array functions and TS complains unless you're very explicit about things.

Unlike `card`, there isn't really a use-case for a `cards` component with one element, so I omitted non-array input from the type signature to keep `ts-lint` from complaining that I always treat it as an array.

Reply:
👍